scripts/generator.js

```diff
@@ -55,8 +56,8 @@ async function renderComponent(options) {
         }
     });
 
-    const encodedProps = encodeURIComponent(JSON.stringify(props));
-    const pageUrl = `${CLIENT_URL}/?component=${component}&props=${encodedProps}&template=${template}`;
+    const encodedProps = qs.stringify({ component, props, template });
+    const pageUrl = `${CLIENT_URL}/?${encodedProps}`;
```

Comment:
Using qs is much more robust IMO, and it yields a valid query string. The previous code didn't produce a valid query string for the stop I was looking at. Puppeteer seemed to accept it though.